README.md

```diff
@@ -12,6 +13,48 @@
 └── <a href="./contracts">contracts</a>: Specular L1 contracts
 </pre>
 
+## Getting Started for Developers
+
+### Development Environment Requirements
+
+To contribute to the Specular monorepo, it will be handy to have the Specular environment and
+toolchain set up. The Specular toolchain depends on:
+
+* go
+* Node with pNPM
+
+These can be installed on macOS with:
+
+```
+> brew install go pnpm
```

Comment:
pnpm should be installed with `corepack enable` after node installation

Reply:
Good to know. I'll have to update my environment. I may figure this out before you respond, but do you know how to install with corepack with brew/apt?

Reply:
NVM, I found instruction and will update my README changes to reflect shortly.